tf_agents/agents/td3/examples/v1/train_eval.py

```diff
@@ -18,12 +18,13 @@
 To run:
 
 ```bash
-tf_agents/agents/td3/examples/v1/train_eval -- \
+tensorboard --logdir $HOME/tmp/td3_v1/gym/HalfCheetah-v2/ --port 2223 &
+
+python tf_agents/agents/td3/examples/v1/train_eval.py \
   --root_dir=$HOME/tmp/td3_v1/gym/HalfCheetah-v2/ \
```

Comment:
$HOME/tmp/td3/v1/gym/HalfCheetah-v2/

Reply:
@sguada I just wanted to make sure I understood this request correctly: you want me to use slashes to separate the algorithm and version number (ex. `td3/v1`) instead of underscores. (ex. `td3_v1`). In this PR I have them all separated by underscores, so should I change them across all algorithms for consistency?

Reply:
td3_v1 is ok, it seems that the other examples use that style.